Parsers/SentinelOne/SentinelOne.txt

```diff
@@ -0,0 +1,277 @@
+                LastLoggedInUserName=column_ifexists('lastLoggedInUserName_s', ''),
+                LicenseKey=column_ifexists('licenseKey_s', ''),
+                LocationEnabled=column_ifexists('locationEnabled_b', ''),
+                LocationType=column_ifexists('locationType_s', ''),
+                Locations=column_ifexists('locations_s', ''),
+                MachineType=column_ifexists('machineType_s', ''),
+                MitigationMode=column_ifexists('mitigationMode_s', ''),
+                MitigationModeSuspicious=column_ifexists('mitigationModeSuspicious_s', ''),
+                ModelName=column_ifexists('modelName_s', ''),
+                NetworkInterfaces=column_ifexists('networkInterfaces_s', ''),
+                NetworkQuarantineEnabled=column_ifexists('networkQuarantineEnabled_b', ''),
+                NetworkStatus=column_ifexists('networkStatus_s', ''),
+                OperationalState=column_ifexists('operationalState_s', ''),
+                OsArch=column_ifexists('osArch_s', ''),
+                DvcOs=column_ifexists('osName_s', ''),
+                OsRevision=column_ifexists('osRevision_s', ''),
+                OsStartTime=column_ifexists('osStartTime_t', ''),
+                OsType=column_ifexists('osType_s', ''),
+                RangerStatus=column_ifexists('rangerStatus_s', ''),
+                RangerVersion=column_ifexists('rangerVersion_s', ''),
+                RegisteredAt=column_ifexists('registeredAt_t', ''),
+                RemoteProfilingState=column_ifexists('remoteProfilingState_s', ''),
+                ScanFinishedAt=column_ifexists('scanFinishedAt_t', ''),
+                ScanStartedAt=column_ifexists('scanStartedAt_t', ''),
+                ScanStatus=column_ifexists('scanStatus_s', ''),
+                ThreatRebootRequired=column_ifexists('threatRebootRequired_b', ''),
+                TotalMemory=column_ifexists('totalMemory_d', ''),
+                UserActionsNeeded=column_ifexists('userActionsNeeded_s', ''),
+                Uuid=column_ifexists('uuid_g', ''),
+                Creator=column_ifexists('creator_s', ''),
+                CreatorId=column_ifexists('creatorId_s', ''),
+                Inherits=column_ifexists('inherits_b', ''),
+                IsDefault=column_ifexists('isDefault_b', ''),
+                Name=column_ifexists('name_s', ''),
+                RegistrationToken=column_ifexists('registrationToken_s', ''),
+                TotalAgents=column_ifexists('totalAgents_d', ''),
+                Type=column_ifexists('type_s', '')
+    | project
+                TimeGenerated, 
+                EventVendor,
+                EventProduct,
+                AccountName,
+                ActivityType,
+                EventCreationTime,
+                DataAccountName,
+                DataFullScopeDetails,
+                DataScopeLevel,
+                DataScopeName,
+                DataSiteId,
+                DataSiteName,
+                UserName,
```

Comment:
Should it not be 'SrcUserName'? Please refer https://docs.microsoft.com/en-us/azure/sentinel/normalization-schema for field names.

Thanks,
Aditi Maheshwari

Reply:
Fixed